.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,3 +24,5 @@ Please delete options that are not relevant.
 - [ ] My code needed automated testing. I have added them (this is optional task)
 
 ## Screenshots (if any)
+
+Please do not use any external image service. Instead, just paste in the image here and it will be uploaded automatically.
```

Comment:
```suggestion
Please do not use any external image service. Instead, just paste in or drag and drop the image here and it will be uploaded automatically.
```